### ISSUE TITLE
Connect wiki_to_netflix/mongo insertion/main file

### ISSUE THREAD

**Issue by audiodude** [noisebridge/MediaBridge#58]
We are matching Wikidata (wiki_to_netflix.py) and now we have code in #55 for inserting movie data into Mongo.

We should reconcile these processes, potentially as part of the (or a...) main file.

**Comment by skyfenton** [comment]
What do we think about how to go about splitting the processing and database loading commands? I was starting to move the processing-related command line tooling into the wiki_to_netflix script now that the main script is integrating multiple parts, but I don't really know how we want to use the command line for different actions.

Should we split it up so that main ONLY runs everything A-Z or can also invoke just processing, loading, etc. commands ex. `main process --full`, `main -v load`? 

I think using main as an entry point to all of our cl tools makes it clearer how to run every option from one --help printout (instead of looking for each entry point in the codebase), but I know we wanted a sort of separation of concerns when it comes to running scripts as `__name__ == __main__`.

**Comment by audiodude** [comment]
It depends on what CLI "processing" we need to do. For the netflix data, we should only really have to do the matching and saving into Mongo once. The only other planned usage of the main script is the CLI MVP.

IMO it's simpler to have subcommands on main rather than to have to dig around and remember which module does which steps. Does typer have so-called "required options"?

**Comment by skyfenton** [comment]
It does have required options, as in options that are required in the command line (besides arguments). I don't think you can specify the logic very complexly (ex. only --full _or_ --num-rows is required), but I doubt we need it.

The only thing that it doesn't quite have are "global" options -- that is, there's not a super easy way to specify a common "-v" flag and implementation across all commands.

> The only other planned usage of the main script is the CLI MVP

Oh right, right. I think I'm going to change the default main.py script behavior (without any other arguments) to do nothing, then, since the main script will be doing many things (maybe `main init`, `main find [movie]`, `main process`).

**Comment by audiodude** [comment]
Yes, in your example I was considering `init` `find` and `process` to be values of the required option `command` or something.
